packages/mux-uploader/src/mux-uploader.ts

```diff
@@ -1,7 +1,27 @@
-import { globalThis } from './polyfills';
+import { globalThis, document } from './polyfills';
+
 import * as UpChunk from '@mux/upchunk';
 
-import rootTemplate from './layouts/block';
+import blockLayout from './layouts/block';
+
+const rootTemplate = document.createElement('template');
+
+rootTemplate.innerHTML = /*html*/ `
+<style>
+  :host {
+    font-family: var(--uploader-font-family, Arial);
+    font-size: var(--uploader-font-size, 16px);
```

Comment:
I think these CSS vars are not really needed because the CSS props can also be set on the custom element itself.
we do this for Media Chrome too for font related props